src/CILogViewer.php

```diff
@@ -525,7 +469,7 @@ private function prepareRawFileName($fileNameInBase64) {
 
         //let's determine what the current log file is
         if(!is_null($fileNameInBase64) && !empty($fileNameInBase64)) {
-            $currentFile = $this->logFolderPath . "/". basename(base64_decode($fileNameInBase64));
+            $currentFile = $this->logFolderPath . basename(base64_decode($fileNameInBase64));
```

Comment:
Are you certain `$this->logFolderPath` is going to end in `/` all the time? I'll say we should do a check if `$this->logFolderPath` ends with a `/` or not

Reply:
You can check that all CI4 paths (APPPATH, SYSTEMPATH, ROOTPATH, ... ) are explicitly defined with a trailing directory separator, and they are concatenated everywhere in the code assuming this, so seems to be kind of the path convention.
However if you want we can add this to make life easier to users.